docroot/modules/humsci/hs_courses/hs_courses.module

```diff
@@ -13,7 +13,27 @@ use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
  */
 function hs_courses_course_collections_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
   // On the ECK entity, if the instructor person is set, use that to display.
-  if (!empty($build['field_instructor_person']['#items'])) {
+  if ($entity->get('field_instructor_person')->count()) {
     unset($build['title']);
+    return;
```

Comment:
You do not want to set the instructor role value if there is an instructor person value?

Reply:
the role gets applied in the field preprocess https://github.com/SU-HSDO/suhumsci/pull/345/files/8699e7e966aebedcdaf441f215fd41a388871882#diff-dc754528fa2ad85d7fdbc30a9c4de508R33